torch/csrc/jit/script/class_type.cpp

```diff
@@ -136,5 +139,15 @@ bool ClassType::isSubtypeOfExt(const TypePtr rhs, std::ostream* why_not) const {
   }
   return Type::isSubtypeOfExt(rhs, why_not);
 }
-
+#else
+bool ClassType::isSubtypeOfExt(const TypePtr rhs, std::ostream* why_not) const {
```

Comment:
This should be implemented by just calling Type::isSubtypeOfExt. This reimplementation will bitrot.

Reply:
If I remove this implementation but keep the declaration with override specifier in the header file, the compiler would complain. Any other idea?

Reply:
Implement this by calling super:

`return Type::isSubtypeOfExt(rhs, why_not);`

Reply:
Got it now. Thanks!